tests/robot-tests/backup-local.py

```diff
@@ -47,10 +47,13 @@
 for blob in generator:
     if '/' in blob.name:
         head, tail = os.path.split(blob.name)
-        os.makedirs(os.path.join(os.getcwd(), 'backup-data', 'content-cache', head), exist_ok=True)
-        block_blob_service.get_blob_to_path('cache', blob.name, os.path.join(os.getcwd(), 'backup-data', 'content-cache', head, tail))
+        os.makedirs(os.path.join(os.getcwd(), 'backup-data',
+                    'content-cache', head), exist_ok=True)
```

Comment:
Did you run the formatter without the `max-line-length` set to 120? Seems like we have all sorts of formatting changes where the lines were below 120 characters.

I'm finding that if I change lines like this to how it was previously, then running the formatter, no changes are made. I'm guessing most of these changes are actually unnecessary.

Would suggest we re-do the formatting and make sure that the max line length has been set correctly to avoid creating lots of potentially incorrect formatting changes.

Reply:
It looks like this was resolved, but it seems like none of the incorrect formatting changes have been reverted

Reply:
I think this should be sorted now